engine/userguide/storagedriver/aufs-driver.md

```diff
@@ -4,248 +4,222 @@ keywords: 'container, storage, driver, AUFS '
+    <truncated output>
+    Storage Driver: aufs
+     Root Dir: /var/lib/docker/aufs
+     Backing Filesystem: extfs
+     Dirs: 0
+     Dirperm1 Supported: true
+    <truncated output>
+    ```
 
-Larger files obviously take longer to *copy up* than smaller files, and files
-that exist in lower image layers take longer to locate than those in higher
-layers. However, a *copy up* operation only occurs once per file on any given
-container. Subsequent reads and writes happen against the file's copy already
-*copied-up* to the container's top layer.
+3.  If you are using a different storage driver, that means that Docker has been
+    explicitly configured to use it. Check `/etc/docker/daemon.json` or the
+    output of `ps auxw |grep dockerd` to see if Docker has been started with
+    the `--storage-driver` flag.
 
-## Deleting files with the AUFS storage driver
 
-The AUFS storage driver deletes a file from a container by placing a *whiteout
-file* in the container's top layer. The whiteout file effectively obscures the
-existence of the file in the read-only image layers below. The simplified
-diagram below shows a container based on an image with three image layers.
+## How the `aufs` storage driver works
 
-![](images/aufs_delete.jpg)
+AUFS is a *union filesystem*, which means that it layers multiple directories on
+a single Linux host and presents them as a single directory. These directories
+are called _branches_ in AUFS terminology, and _layers_ in Docker terminology.
+The unification process is referred to a a _union mount_.
 
-The `file3` was deleted from the container. So, the AUFS storage driver  placed
-a whiteout file in the container's top layer. This whiteout file effectively
-"deletes" `file3` from the container by obscuring any of the original file's
-existence in the image's read-only layers. This works the same no matter which
-of the image's read-only layers the file exists in.
+The diagram below shows a Docker container based on the `ubuntu:latest` image.
 
-## Renaming directories with the AUFS storage driver
+![Layers of an Ubuntu container](images/aufs_layers.jpg)
 
-Calling `rename(2)` for a directory is not fully supported on AUFS. It returns
-`EXDEV` ("cross-device link not permitted"), even when both of the source and
-the destination path are on a same AUFS layer, unless the directory has no
-children.
+Each image layer, and the container layer, are represented on the Docker host as
+subdirectories within `/var/lib/docker/`. The union mount provides the unified
+view of all layers. The directory names do not directly correspond to the IDs
+of the layers themselves.
 
-So your application has to be designed so that it can handle `EXDEV` and fall
-back to a "copy and unlink" strategy.
+AUFS uses the Copy-on-Write (CoW) strategy to maximize storage efficiency and
+minimize overhead.
 
-## Configure Docker with AUFS
+### Example: Image and container on-disk constructs (`overlay`)
```

Comment:
use of `overlay` as a term may be confusing here in this context (as this is the `aufs` page)